contents/docs/deployment/deploy-aws-clickhouse.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Deploying to AWS with Clickhouse
+sidebar: Docs
+showTitle: true
+---
+
+## Why clickhouse on AWS?
+
+This deployment mechanism is targeted towards larger installations (above 2M events/month) which would run into performance and scalability issues using our normal postgres-based deployments by using the highly scalable clickhouse database.
```

Comment:
Thanks! We'll need to adjust this a bit since we'll likely be open-sourcing clickhouse as well at some point.

Aside: One question that needs answering eventually is how to call this sort of deployment in a clean way? self-hosted clickhouse? ...?